tests/run-pass/2phase.rs

```diff
@@ -1,4 +1,4 @@
-#![feature(nll)]
+// edition:2018
```

Comment:
All our tests are run in the 2018 edition, so this comment should not be needed.

Reply:
Removed